### PR TITLE
Document the new options introduced in https://github.com/qgis/qwc2/pull/73

### PR DESCRIPTION
**Pull request by autra**
This PR documents the proposed change in https://github.com/qgis/qwc2/pull/73

### CODE REVIEW

**Comment by autra** [comment]
Should I update he submodule commit hash here once https://github.com/qgis/qwc2/pull/73 is merged?

File: themesConfig.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
     "items": [
       {
         "url": "http://qwc2.sourcepole.ch/wms/wolfsburg/qwc2_demo",
+        "wmsBasicAuth": {
+          "username": "test",
+          "password": "test"
+        },
```
Comment (manisandro) [inline themesConfig.json]: I'd leave this out, since it is kinda misleading as this server does not require authentication.

**Comment by manisandro** [comment]
I can update the submodule, I'm updating it fairly often at the moment anyways.

File: themesConfig.json (hunk 1 above)
Comment (autra) [reply themesConfig.json]: Oh yes, sorry, this was not meant to be included.

**Comment by autra** [comment]
@manisandro updated :-)

**Comment by manisandro** [comment]
Thanks!

**Pull request merged**